prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -344,6 +345,17 @@ public async Task SendPaperEnrolmentSubmissionEmailAsync(int enrolleeId)
             await Send(email);
         }
 
+        public async Task SendEnrolleeAbsenceNotificationEmailAsync(int enrolleeId, EnrolleeAbsenceViewModel absence, string email)
+        {
+            var viewModel = await _context.Enrollees
+                .Where(e => e.Id == enrolleeId)
+                .Select(e => new EnrolleeAbsenceNotificationEmailViewModel(e.FirstName, e.LastName, absence.StartTimestamp, absence.EndTimestamp))
```

Comment:
Could you test what SQL gets generated by this? This is the first time in the app we are using the _constructor_ of a view model rather than assigning the properties using object initialization syntax a la
```c#
.Select(e => new EnrolleeAbsenceNotificationEmailViewModel
{
  e.FirstName, 
  e.LastName, 
  absence.StartTimestamp, 
  absence.EndTimestamp  
})
```
EF might be smart enough here but what may happen is that the entire `Enrollee` model is retrieved from the DB and then just `FirstName` and `LastName` get mapped into the VM object.

Reply:
It appears to be smart enough to not grab it all 👍 
```
SELECT e."FirstName", e."LastName"
FROM "Enrollee" AS e
WHERE e."Id" = @__enrolleeId_0
LIMIT 2
```